tests/testthat/test-shinytest2-tm_front_page.R

```diff
@@ -0,0 +1,48 @@
+test_that("e2e: tm_front_page initializes without errors", {
+  skip_if_too_deep(5)
+  require(shinytest2)
+
+  data <- simple_cdisc_data()
+  data <- within(data, {
+    attr(ADSL, "metadata") <- list("Author" = "NEST team", "data_source" = "synthetic data")
+  })
+
+  app <- TealAppDriver$new(
```

Comment:
```suggestion
  app <- teal::TealAppDriver$new(
```

Reply:
`TealAppDriver` is internal class .  we have import non-exported `TealAppDriver `from `teal` package in  helper-TealAppDriver.R

Reply:
ah, just seeing this in `tests/testthat/helper-TealAppDriver.R`